src/riak_test_escript.erl

```diff
@@ -149,6 +149,7 @@ main(Args) ->
                     Tests0;
                 {_, undefined} ->
                     Tests0;
+                %% TODO don't understand what this does
```

Comment:
Not sure where that is from. I need to rebase this, it will probably disappear then.